internal/cli/config/config.go

```diff
@@ -57,7 +57,7 @@ func (opts *configOpts) initStore() error {
 }
 
 func (opts *configOpts) IsCloud() bool {
-	return opts.Service == config.CloudService
+	return opts.Service == config.CloudService || opts.Service == config.CloudGovService
```

Comment:
this may work for now, but not all services are available to gov, data lake is one of them but we can refine this later

Reply:
if the service is not supported in gov, couldn’t we just not add `config.CloudGovService` to the function in the store?

Reply:
I mean in the switch inside the store file:
```
func (s *Store) Projects(opts *atlas.ListOptions) (interface{}, error) {
	switch s.service {
	case config.CloudService, config.CloudGovService:
		result, _, err := s.client.(*atlas.Client).Projects.GetAllProjects(context.Background(), opts)
```

If a service is not implemented for gov we could just remove `config.CloudGovService ` from the related function in the store.

Reply:
yep, something like that may be the way forward we just need to verify what's available and what not and that may take some time

Reply:
resolving this for now